ui/src/app/common/store/actions/user.actions.ts

```diff
@@ -0,0 +1,17 @@
+import _ from 'lodash';
+import { Store } from '@ngrx/store';
+import { UserService } from '../../services';
+import { TFullUserInfo } from '../../types';
+import { TStoreCommonModule } from '../index';
+import { setUserInfo } from '../reducers/user.reducer';
+
+export const loadUserInfo = (
+	userService: UserService,
+	store: Store<TStoreCommonModule>,
+	callback: () => void = _.noop
```

Comment:
Using callbacks in actions - is not a good way you'll get a mess in the result. You can only dispatch actions

Actually, it's better to use effects for BE interaction. https://habr.com/ru/post/448858/

Reply:
https://github.com/taraswww777/AngularGlobalMentoringProgram2019/commit/5ecfbac7